### DISCUSSION TITLE
posts/2021-12-31-npm-package

### DISCUSSION THREAD

**Discussion by giscus** [cmdcolin/cmdcolin.github.io#11] [General]
# posts/2021-12-31-npm-package

Blogging for the future

https://cmdcolin.github.io/posts/2021-12-31-npm-package

**Comment by mahdikhashan** [comment]
thanks for sharing

**Comment by AdelBeit** [comment]
super helpful!

**Comment by jackchoumine** [comment]
how to generate docs according ts type?

**Reply by cmdcolin** [reply]
I have used the "documentation" npm package to do this. example of docs that are autogenerated into a readme file https://github.com/gmod/cram-js?tab=readme-ov-file#api-auto-generated

https://www.npmjs.com/package/documentation
